### PR TITLE
SauceLabs UX test will use env.APP_URL

### PR DESCRIPTION
**Pull request by sdevopsadm**
* Fixes sdevopsadm/roadmap#37

### CODE REVIEW

File: spring-petclinic-api-gateway/src/test/resources/sample.js [hunk 1]
```diff
@@ -23,10 +23,11 @@ console.log('hello')
 console.log(process.env.SAUCE_USERNAME)
 console.log(process.env.SAUCE_APIKEY)
 
+var URL = process.env.APP_URL;
```
Comment (jgkong) [inline spring-petclinic-api-gateway/src/test/resources/sample.js]: That's good idea to separate variable assign.

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-062956 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170525-062956") to [POC](https://console.ng.bluemix.net/apps/0e2c13c4-54ec-45ed-9274-834bec850d52?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-094810 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-094810") to [POC](https://console.ng.bluemix.net/apps/e0ff7bdd-8955-4b8a-81ba-cc3bcda0f41d?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-101114 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170607-101114") to [POC]( "ibm:yp:us-south:VDInfraPOC:POC"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-POC-20170614-115903 "deploy-ibm-yp-us-south-VDInfraPOC-POC-20170614-115903") to [POC](https://console.ng.bluemix.net/apps/0e2c13c4-54ec-45ed-9274-834bec850d52?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:POC"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-041513 "deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-041513") to [dev](https://console.bluemix.net/apps/a4a4c98c-f00c-41f5-a60c-4751e5da00e0?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:dev"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-042646 "deploy-ibm-yp-us-south-VDInfraPOC-dev-20170629-042646") to [dev](https://console.bluemix.net/apps/076deb48-e8d0-4c56-b906-7cca0e7090da?env_id=ibm:yp:us-south "ibm:yp:us-south:VDInfraPOC:dev"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5e65a470-ac27-4ff4-9123-1ce97f9f44db "PoC2-api-gateway-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-043922 "deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-043922") to [qa]( "ibm:yp:us-south:VDInfraPOC:qa"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Comment by sdevopsadm** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/85559b6d-3b47-4c86-88b5-a3688e68285b "PoC2_petclinic_microservice_jgkong"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/673651ec-45f6-4e57-977b-ac018a92f419 "PoC2-vets-service-jgkong") deployed [spring-petclinic-microservices](https://github.com/sdevopsadm/spring-petclinic-microservices/releases/tag/deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-051215 "deploy-ibm-yp-us-south-VDInfraPOC-qa-20170629-051215") to [qa]( "ibm:yp:us-south:VDInfraPOC:qa"), including commit bdbd68899778210e635b73c04c52496266ffd3a1

**Pull request merged**